testing/scenariobuilder/builder.go

```diff
@@ -174,6 +175,12 @@ func (s *TodocheckScenario) ExpectExecutionError() *TodocheckScenario {
 	return s
 }
 
+// ExpectJsonOutput sets the output to be in JSON format
+func (s *TodocheckScenario) ExpectJsonOutput() *TodocheckScenario {
```

Comment:
I think this should be `WithJSONOutput` instead (sorry, I know I suggested the name initially :) ), because it changes the input of the program AND the expected output.